app/modes/library/book_cover.py

```
@@ -0,0 +1,211 @@
+        return json.loads(urllib.request.urlopen(req, timeout=30).read())
+
+    def _get_image(self, filename, subfolder, folder_type):
+        """Download a generated image from ComfyUI."""
+        params = {"filename": filename, "subfolder": subfolder, "type": folder_type}
+        url_values = urllib.parse.urlencode(params)
+        with urllib.request.urlopen(
+            f"http://{self.server_address}/view?{url_values}", timeout=60
+        ) as response:
+            return response.read()
+
+    def _load_workflow(self):
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `_load_workflow`

(ANN202)